src/main/java/Sally.java

```diff
@@ -0,0 +1,291 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Sally {
+    protected static ArrayList<Task> list = new ArrayList<>();
+    private static Scanner sc;
+
+    public static void main(String[] args) {
+        printBorder();
+        System.out.println("Hello! I'm Sally");
+        System.out.println("What can I do for you?");
+        printBorder();
+
+        try {
+            readsFile();
+        } catch (FileNotFoundException e) {
+        }
+        sc = new Scanner(System.in);
+        messaging();
+    }
+
+    public static void readsFile() throws FileNotFoundException {
+        String filePath = "D:/NUS/Y2/S1/CS2103T/ip/data/Sally.txt";
```

Comment:
Consider making `filePath` system-agnostic. Also, other users who run your app may not have a D drive.